docs/index.asciidoc

```diff
@@ -374,8 +374,8 @@ default to the current OS temporary directory in linux /tmp/logstash
 
 Set the time, in MINUTES, to close the current sub_time_section of bucket.
 If you define file_size you have a number of files in consideration of the section and the current tag.
-0 stay all time on listener, beware if you specific 0 and size_file 0, because you will not put the file on bucket,
-for now the only thing this plugin can do is to put the file when logstash restart.
+If you also define file_size you have a number of files related to the section and the current tag.
+If it's valued 0 and `rotation_strategy` is `time` or `size_and_time` then the plugin reaise a configuration error.
```

Comment:
```suggestion
 If <<plugins-{type}s-{plugin}-rotation_strategy,`rotation_strategy`>> is set to `time` or `size_and_time`, then `time_file` cannot be set to 0. 
 Otherwise, the plugin raises a configuration error.
```

Reply:
Since you like the wording, I put it in as a suggestion with a link back to `rotation_strategy`.  I removed the "Some settings are not compatible" part after you helped me get a better understanding.  

Please take a look at Line 376 in the original file and Line 377 in your changes.  It looks like is still in the original file, but also repeated in your changes.

Reply:
Thanks a lot @karenzone for the good reword, I integrated into the PR